packages/playground/src/app/index.tsx

```diff
@@ -1,17 +1,22 @@
 import { Console } from './Console';
-import { InkEditor, exampleCode } from '@paritytech/ink-editor';
+import { InkEditor } from '@paritytech/ink-editor';
 import { Layout } from './Layout';
 import { Header } from './Header';
 import { AppContext, AppProvider } from '~/context/app/';
 import { MessageContext, MessageProvider } from '~/context/messages/';
-import { ReactElement, useContext, useState } from 'react';
+import { ReactElement, useContext, useEffect, useState } from 'react';
 import { Dispatch, State } from '~/context/app/reducer';
 import { MessageDispatch, MessageState } from '~/context/messages/reducer';
+import { loadCode } from '~/context/side-effects/load-code';
 
 const App = (): ReactElement => {
-  const [code, setCode] = useState(exampleCode);
   const [state, dispatch]: [State, Dispatch] = useContext(AppContext);
   const [, messageDispatch]: [MessageState, MessageDispatch] = useContext(MessageContext);
+  const [code, setCode] = useState<string>();
+
+  useEffect(() => {
+    loadCode(state, { app: dispatch, message: messageDispatch }).then(setCode);
```

Comment:
Load code returns either the example code or if there was a params id, the code from gist.